src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURE_ARRAY_SIZE = 6;

```

Comment:
According to the checklist, magic numbers in the code should be constants. However, the name 'FIGURE_ARRAY_SIZE' is not descriptive enough to understand what kind of figures this size refers to. Consider renaming it to something more specific, like 'DEFAULT_FIGURES_ARRAY_SIZE', to clarify its purpose.